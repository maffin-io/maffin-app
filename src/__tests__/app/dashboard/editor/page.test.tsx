import { render } from '@testing-library/react';
import React from 'react';

import EditorPage from '@/app/dashboard/editor/page';

jest.mock('@/components/SQLEditor', () => jest.fn(
  () => <div data-testid="SQLEditor" />,
));

jest.mock('@/components/SQLExplorer', () => jest.fn(
  () => <div data-testid="SQLExplorer" />,
));

describe('EditorPage', () => {
  it('renders as expected', () => {
    const { container } = render(<EditorPage />);

    expect(container).toMatchSnapshot();
  });
});
