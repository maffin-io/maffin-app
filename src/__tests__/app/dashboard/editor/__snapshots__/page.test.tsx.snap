// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditorPage renders as expected 1`] = `
<div>
  <div
    class="grid grid-cols-12 items-start"
  >
    <div
      class="col-span-9"
    >
      <div
        data-testid="SQLEditor"
      />
    </div>
    <div
      class="col-span-3 card"
    >
      <h2>
        Tables
      </h2>
      <div
        data-testid="SQLExplorer"
      />
    </div>
  </div>
</div>
`;
